docs/reference/commandline/stats.md

```diff
@@ -22,6 +22,7 @@ Display a live stream of container(s) resource usage statistics
 
 Options:
   -a, --all         Show all containers (default shows just running)
+      --format string   Pretty-print images using a Go template
```

Comment:
can you update the other lines as well, so that it aligns (as output on the command line?)